lesson02/canDrive.js

```diff
@@ -0,0 +1,25 @@
+const firstName = "Oyun";
+const lastName = "Batnasan";
+const age = 29;
+const drivingAge = 16;
+const drivingTime = age - drivingAge;
+
+
+//check to see if I'm illegal to drive
+//if I'm prin Oyun Batnasan can drive and has driven for x years
+if (age >= drivingAge) {
+  //do codes
```

Comment:
heh...that lingo looks like my lingo. Make sure you're not just copying my demo. :)